### ISSUE TITLE
warning: Didn't have any audio data in callback (buffer underflow) on MacOS High Sierra

### ISSUE THREAD

**Issue by nospam2k** [audiojs/audio-play#15]
After playing a buffer, I get the following error.

play('testfile.mp3');

[../deps/mpg123/src/output/coreaudio.c:81] warning: Didn't have any audio data in callback (buffer underflow)

This error repeats in a loop.

**Comment by connorhartley** [comment]
This should be fixed with the custom speaker implementation. https://github.com/audiojs/audio-speaker/pull/44

**Comment by getsbydev** [comment]
Any progress so far? Still getting this warning...

**Comment by tho-cograph** [comment]
I got the same issue, how to stop warning
_[../deps/mpg123/src/output/coreaudio.c:81] warning: Didn't have any audio data in callback (buffer underflow)_

**Comment by mindblower23** [comment]
I also see this warning! Any work around yet?

**Comment by dy** [comment]
We're blocked by https://github.com/audiojs/node-cubeb, https://github.com/audiojs/audio-speaker/pull/44 or https://github.com/audiojs/audio-mpg123/pull/10.
That needs a separate effort.
